features/migrate_client_to_minion.feature

```diff
@@ -2,6 +2,9 @@
 # Licensed under the terms of the MIT license.
 
 Feature: Migrate a traditional client into a salt minion
+  In order to move away from traditional clients
```

Comment:
i think we can remove this line. the two after are enough and describe better the feature.